code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -108,7 +109,15 @@
 			C.apply_damage(5, BRUTE, E)
 			user.spin(20, 1)
 			playsound(user.loc, 'sound/weapons/slash.ogg', 50, 0)
-
+			add_logs(user, C, "tail whipped")
+			if(user.restrained())
+				if(prob(50))
+					user.Weaken(5)
+					user.visible_message("<span class='danger'>[src] lost their balance! </span>", "<span class='danger'>You lost your balance!</span>")
```

Comment:
Change this to `"<span class='danger'>[src] loses [p_their()] balance!</span>", "<span class='danger'>You lose your balance!</span>"

Reply:
New pronouns ftw